lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -1080,6 +1080,10 @@ CompilerType SwiftLanguageRuntimeImpl::GetChildCompilerTypeAtIndex(
     LLDBTypeInfoProvider tip(*this, *instance_ts);
     reflection_ctx->ForEachSuperClassType(
         &tip, pointer, [&](SuperClassType sc) -> bool {
+          // If the typeref is invalid, we don't want to process it.
+          if (!sc.get_typeref()) 
+            return false;
```

Comment:
Is this an error, or expected? We might want to log it if it's an error.

Reply:
If it's expected, because this is normal for artificial base classes, maybe the comment could be more explicit about that?

Reply:
It's expected if the current class is artificial, I'll add a comment

Reply:
Remote Mirrors doesn't really tell us why getting the typeref failed (it could be an artificial class or conceivably something else). It'd be pretty straight forward to add an API asking if the class is artificial or not, if we want that.